.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+  push:
+    branches: master
+
+name: Trdelnik tests
+
+env:
+  SOLANA_CLI_VERSION: 1.10.8
+
+jobs:
+  turnstile:
+    runs-on: ubuntu-18.04
+    defaults:
+      run:
+        working-directory: examples/turnstile
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup-solana/
+      - name: Install depends
+        run: |
+          sudo apt-get update && sudo apt-get install -y build-essential libudev-dev
+          rustup default nightly
+          rustup component add rustfmt
+          cargo install cargo-expand
+          cargo install --force cargo-make
+      - name: Cargo build
+        run: cargo build
+      - name: Prepare program client
+        run: makers trdelnik test
```

Comment:
Is it necessary? We have this command two times there (bellow).

Reply:
First time it will create `program_client` second time it will run tests

Reply:
ok so it should be part of the build command or something like this in the future?

Reply:
Yes exactly, once this [PR](https://github.com/Ackee-Blockchain/trdelnik/pull/21) will be rebased and merged